lib/ui/widgets/login_google_tile.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:pispapp/ui/widgets/title_text.dart';
+
+class LoginWithGoogleTile extends StatelessWidget {
+  const LoginWithGoogleTile({Key key, this.trailingWidget}) : super(key: key);
+
+  final Widget trailingWidget;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListTile(
```

Comment:
Would it be possible to automatically go to the next page once the login is successful? Because currently we cannot differentiate when login is not successful and there is no text that display the info as well

Reply:
you get a circular tick on the top right

Reply:
Hmm... I think we may want to improve the flow in the future, let's keep this open for reference